section1/reflection.md

```diff
@@ -1,19 +1,37 @@
 ## Section 1 Reflection
```

Comment:
Great work in Section 1 and I appreciate your reflection.

That first point covered in Q1 is huge - yup. That is REAL for a lot of us! Keep an eye on it, use your strategies and know you are not alone - hopefully your cohort grows a culture where everyone can feel they can ask those questions, or share their tummy flips.